images/git/alpine.tf

```diff
@@ -6,11 +13,9 @@ module "latest-alpine" {
   source            = "../../tflib/publisher"
   name              = basename(path.module)
   target_repository = var.target_repository
-  config            = file("${path.module}/configs/latest.alpine.${each.key}.apko.yaml")
+  config            = module.alpine[each.key].config
   check-sbom        = false # Alpine-based SBOMs are not conformant because the Alpine baselayout has an invalid license specifier
-  # Override the module's default wolfi packages that conflict with alpine
-  extra_packages = []
```

Comment:
Do we not still need this?

Reply:
The default is `[]` already, it shouldn't have any effect (please fact check)

Reply:
You are right, this must predate our use of the provider for this.